packages/linter/src/utils/convert-tslint-to-eslint/convert-to-eslint-config.ts

```diff
@@ -188,3 +189,49 @@ export async function convertToESLintConfig(
     ),
   };
 }
+
+function likelyContainsTSLintComment(fileContent: string): boolean {
+  return fileContent.includes('tslint:');
+}
+
+function allFilesInDirInTree(tree: Tree, dir: string): string[] {
+  let files: string[] = [];
+  tree.children(dir).forEach((child) => {
+    const childPath = normalizePath(join(dir, child));
+    if (tree.isFile(childPath)) {
+      files.push(childPath);
+      return;
+    }
+    files = [...files, ...allFilesInDirInTree(tree, childPath)];
+  });
+  return files;
+}
+
+export function convertTSLintDisableCommentsForProject(
+  tree: Tree,
+  projectName: string
+) {
+  /**
+   * We need to avoid a direct dependency on tslint-to-eslint-config
+   * and ensure we are only resolving the dependency from the user's
+   * node_modules on demand (it will be installed as part of the
+   * conversion generator).
+   */
+  const { convertFileComments } = getConvertToEslintConfig();
+
+  const { root } = readProjectConfiguration(tree, projectName);
+  const allSourceFiles = allFilesInDirInTree(tree, root);
```

Comment:
We can use [`visitNotIgnoredFiles`](https://github.com/nrwl/nx/blob/01967f89b1b281750ae74c742ac2be12aa881c5e/packages/devkit/src/generators/visit-not-ignored-files.ts#L10)